org.osgi.test.assertj.cm/src/main/java/org/osgi/test/assertj/cm/configurationevent/ConfigurationEventType.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.cm.configurationevent;
+
+import static org.osgi.service.cm.ConfigurationEvent.CM_DELETED;
+import static org.osgi.service.cm.ConfigurationEvent.CM_LOCATION_CHANGED;
+import static org.osgi.service.cm.ConfigurationEvent.CM_UPDATED;
+
+import org.osgi.test.common.bitmaps.Bitmap;
+
+public class ConfigurationEventType {
+
+	public static final int[] TYPES = {
```

Comment:
This is dangerous since the array is mutable. This should be a list or set. Or private.

Reply:
okay . private.

but we use dame pattern in FrameworkEventType,BundleEventType,BundleState and ServiceEventType